tests/src/test/java/alluxio/client/fs/LocalFirstPolicyIntegrationTest.java

```diff
@@ -107,22 +107,6 @@ public void test() throws Exception {
       assertEquals(100, blockWorker1.getStoreMeta().getUsedBytes());
       assertEquals(0, blockWorker2.getStoreMeta().getUsedBytes());
     }
-
```

Comment:
pls move this to a separate `@Test` method in the same class, instead of a new one